app/api/v1/users.py

```
@@ -27,3 +32,24 @@ def create_user(payload: CreateUserRequest, uc: CreateUser = Depends(get_create_
+    uc: GetMe = Depends(get_get_me_uc),
+) -> UserResponse:
+    user = uc.execute(user_id)
+    if not user:
+        raise HTTPException(status_code=404, detail="User not found")
+    return UserResponse.model_validate(user)
+
+@router.patch("/me", response_model=UserResponse)
+async def update_me(
+    payload: UpdateMeRequest,
+    user_id: str = Depends(get_current_user_id),
+    uc: UpdateMe = Depends(get_update_me_uc),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)